tgui/packages/tgui/interfaces/Safe.js

```diff
@@ -0,0 +1,149 @@
+import { Fragment } from 'inferno';
+import { resolveAsset } from '../assets';
+import { useBackend } from '../backend';
+import { Box, Button, Icon, Section } from '../components';
+import { Window } from "../layouts";
+
+export const Safe = (properties, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    dial,
+    open,
+  } = data;
+  return (
+    <Window
+      width={625}
+      height={750}
+      theme="safe">
+      <Window.Content>
+        <Box className="Safe--engraving">
+          <Dialer />
+          <Box>
+            <Box
+              className="Safe--engraving--hinge"
+              top="25%" />
+            <Box
+              className="Safe--engraving--hinge"
+              top="75%" />
+          </Box>
+          <Icon
+            className="Safe--engraving--arrow"
+            name="long-arrow-alt-down"
+            size="3"
+          /><br />
+          {open ? (
+            <Contents />
+          ) : (
+            <Box
+              as="img"
+              className="Safe--dial"
+              src={resolveAsset('safe_dial.png')}
+              style={{
+                "transform": "rotate(-" + (3.6 * dial) + "deg)",
+              }}
+            />
+          )}
+        </Box>
+        {!open && (
+          <Help />
+        )}
+      </Window.Content>
+    </Window>
+  );
+};
+
+const Dialer = (properties, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    dial,
+    open,
+    locked,
+    broken,
+  } = data;
+  const dialButton = (amount, right) => {
+    return (
+      <Button
+        disabled={open || (right && !locked) || broken}
+        icon={"arrow-" + (right ? "right" : "left")}
+        content={(right ? "Right" : "Left") + " " + amount}
+        iconRight={right}
+        onClick={() => act(!right ? "turnright" : "turnleft", {
+          num: amount,
+        })}
+      />
+    );
+  };
+  return (
+    <Box className="Safe--dialer">
+      <Button
+        disabled={locked && !broken}
+        icon={open ? "lock" : "lock-open"}
+        content={open ? "Close" : "Open"}
+        mb="0.5rem"
+        onClick={() => act('open')}
+      /><br />
+      <Box position="absolute">
+        {[dialButton(50), dialButton(10), dialButton(1)]}
+      </Box>
+      <Box
+        className="Safe--dialer--right"
```

Comment:
Same for these, `Safe__dialerRight`, or `Safe__dialer-right`

Reply:
Renamed.